### PR TITLE
Submit message cleanup

### LINKED ISSUE #3323: Cleanup of submit-message page
Some design changes to the submit-message page.

large-screen-submit-message
Changes:
- [ ] Change to page title/alert design
- [ ] Adjustment to "receipt" style and layout
    - [ ] Relinquishment section only appears when Chapter 33 has been selected
- [ ] Document statement bundled into accordion
- [ ] Remove blue "main page" button
* Design doesn't incorporate breadcrumbs, but it should

large-screen-submit-message-accordion-open
Changes:
- [ ] Shows accordion treatment of document details

small-screen-submit-message-accordion-open
Changes:
- [ ] Small screen version of items noted above. Accordion shown open for reference only. Defaults to closed.

### PR DESCRIPTION
**Pull request by alexose**
Fixes #3323 .

This also adds some capabilities to the ExpandingGroup component.

### CODE REVIEW

**Comment by akainic** [comment]
Can you please include screenshots? Was having trouble running this locally and it would help if I could review how it looks. Thanks!

**Comment by alexose** [comment]
Screenshots as requested:

image

Document box expanded:

image

**Comment by alexose** [comment]
It seems like we should hide the "relinquished" box if the user doesn't relinquish anything.  Or is it better to be explicit, and just use different words?

**Comment by goldenmeanie** [comment]
We should show what they chose in that part of the form. But we don't show anything if they aren't dealing with Post-9/11.

File: src/js/common/helpers/environment.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const _Environments = {
   staging: { API_URL: 'https://staging-api.vets.gov', BASE_URL: 'https://staging.vets.gov' },
   production: { API_URL: 'https://dev-api.vets.gov', BASE_URL: 'https://dev.vets.gov' },
-  development: { API_URL: 'http://localhost:3000', BASE_URL: 'http://localhost:3001' },
```
Comment (jbalboni) [inline src/js/common/helpers/environment.js]: This is used by other apps, too. I'm not sure we want to change it.

File: src/js/edu-benefits/containers/SubmitPage.jsx [hunk 2]
```diff
@@ -7,28 +7,22 @@ import { relinquishableBenefits } from '../utils/options-for-select';
...
-          address={address}
-          claimedBenefits={benefits}
-          relinquishedBenefits={relinquished}/>
+      <SubmitMessage claimType="Education Benefits" {...this.props}/>
     );
   }
 }
 
 function mapStateToProps(state) {
+  // TODO: replace this with actual data from the API
```
Comment (jbalboni) [inline src/js/edu-benefits/containers/SubmitPage.jsx]: I don't think we're planning on more data from the api.

**Review by jbalboni — APPROVED** [review]
Looks good to me, main question I had was around the environment.js file.

File: src/js/edu-benefits/containers/SubmitPage.jsx (hunk 2 above)
Comment (alexose) [inline src/js/edu-benefits/containers/SubmitPage.jsx]: It seems a little weird to me that we populate it with data from the client.  I suppose it's fine-- the UX doesn't change either way.

Comment removed.

**Comment by alexose** [comment]
This isn't passing on dev because a unit test assumes a successful submit.  In retrospect, that makes it an integration test... I'll rework it so it makes more sense.

**Pull request merged**